packages/benchmark/src/benchmarkerator.js

```diff
@@ -90,6 +100,22 @@ import { makeWalletFactoryDriver } from '@agoric/boot/tools/drivers.ts';
  *   // benchmark context parameter of subsequent calls into the benchmark object.
  *   setup?: (context: BenchmarkContext) => Promise<Record<string, unknown> | undefined>,
  *
+ *   // Optional setup method for an individual round.  This is executed as part
```

Comment:
I've been frustrated (in a way I think we've discussed previously) by the need to, in essence, write the declarations twice: once for the type checker, using one syntax, and once for the human reader, in a different syntax to provide documentation text that the JS type syntax did not permit.  Since the benchmarks already have to be executed using `tsx` anyway, we're already in the TS world here, so I think converting this to TS may be the best course.  If you're game for that I'm happy to have you do it, but I'd prefer to do it in a later PR as I've already been burned a couple times by TS conversions happening in the middle of trying to do something else unrelated.

Reply:
Good call. A full conversion could wait for another PR. I just pushed a commit to solve this specific duplicate issue by extracting the types to a `types.ts`.